x-pack/test/security_solution_endpoint/apps/endpoint/index.ts

```diff
@@ -16,8 +16,7 @@ import {
 export default function (providerContext: FtrProviderContext) {
   const { loadTestFile, getService, getPageObjects } = providerContext;
 
-  // FLAKY: https://github.com/elastic/kibana/issues/180401
-  describe.skip('endpoint', function () {
```

Comment:
Hey @gergoabraham We saw similar flakiness before and from appex-qa perspective it is related to basic authentication in UI tests (line 42 calls `svlCommonPage.login()`) for serverless env when serverless env is setup with mock-idp to do SAML authentication. I tried retries before myself and flaky-test-runner proved it won't help as redirects issue still persist.

The right solution is to migrate the FTR tests to role-based SAML authentication, it is mostly 1 line change and we didn't see similar flakiness:
https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/page_objects/svl_common_page.ts#L73-L130
More details with examples are [here](https://github.com/elastic/kibana/tree/main/x-pack/test_serverless#roles-based-testing) 

You probably saw recent email about new preferred login method for a local serverless Kibana, with that we consider deprecating `svlCommonPage.login()` as all e2e UI tests should be using SAML auth.

Reply:
hey @dmlemeshko, a big thank you for letting me know! 🙌 

addressing the change here:
- https://github.com/elastic/kibana/pull/189248